tests/acceptance/config/behat.yml

```diff
@@ -8,128 +8,127 @@ default:
         - '%paths.base%/../features/apiPasswordAddUser'
       contexts:
         - PasswordPolicyContext:
-        - OccContext:
         - FeatureContext: &common_feature_context_params
             baseUrl:  http://localhost:8080
             adminUsername: admin
             adminPassword: admin
             regularUserPassword: 123456
             ocPath: apps/testing/api/v1/occ
+        - OccContext:
 
     apiPasswordChange:
       paths:
         - '%paths.base%/../features/apiPasswordChange'
       contexts:
         - PasswordPolicyContext:
-        - OccContext:
         - FeatureContext: *common_feature_context_params
+        - OccContext:
 
     apiUpdateShare:
       paths:
         - '%paths.base%/../features/apiUpdateShare'
       contexts:
         - PasswordPolicyContext:
-        - OccContext:
         - FeatureContext: *common_feature_context_params
+        - OccContext:
 
     apiGuests:
       paths:
         features: '%paths.base%/../features/apiGuests'
       contexts:
         - PasswordPolicyContext:
-        - FeatureContext: *common_feature_context_params
         - EmailContext:
+        - FeatureContext: *common_feature_context_params
         - GuestsContext:
 
     cliPasswordAddUser:
       paths:
         - '%paths.base%/../features/cliPasswordAddUser'
       contexts:
         - PasswordPolicyContext:
+        - FeatureContext: *common_feature_context_params
         - OccContext:
         - OccUsersGroupsContext:
-        - FeatureContext: *common_feature_context_params
 
     cliPasswordChange:
       paths:
         - '%paths.base%/../features/cliPasswordChange'
       contexts:
         - PasswordPolicyContext:
+        - FeatureContext: *common_feature_context_params
         - OccContext:
         - OccUsersGroupsContext:
-        - FeatureContext: *common_feature_context_params
 
     webUIPasswordAddUser:
       paths:
         - '%paths.base%/../features/webUIPasswordAddUser'
       contexts:
         - PasswordPolicyContext:
-        - WebUIPasswordPolicyContext:
-        - WebUIGeneralContext:
-        - WebUILoginContext:
         - EmailContext:
+        - FeatureContext: *common_feature_context_params
         - WebUIFilesContext:
+        - WebUIGeneralContext:
+        - WebUILoginContext:
+        - WebUIPasswordPolicyContext:
         - WebUIPersonalGeneralSettingsContext:
-        - WebUIUsersContext:
```

Comment:
This exists in core ``stable10`` and ``webUIPasswordAddUser`` suite is just run against core ``stable10`` so it can test user_management stuff with password_policy.
Some day if user_management is separated out as a separate app in ``stable10`` then this will need to be reviewed.
But for now this is the way it should work.